providers/shared/attributesets/event/id.ftl

```diff
@@ -0,0 +1,38 @@
+[#ftl]
+
+[@addAttributeSet
+    type=EVENT_ATTRIBUTESET_TYPE
+    properties=[
+        {
+                "Type"  : "Description",
+                "Value" : "Configuration of reported system notifications from component resources"
+        }]
+    attributes=[
+        {
```

Comment:
These are very specific to DB components. I think it would be better that this was on the DB component. If you think it should be an attribute set. EVENT_ATTRIBUTE_SET_TYPE is to generic. 

Maybe DB_SYSTEM_EVENT_ATTRIBUTE_TYPE would be better.

Reply:
Done